src/clients/api/queries/getVTokenApySimulations/index.ts

```diff
@@ -56,6 +59,30 @@ const getVTokenApySimulations = async ({
     });
   }
 
+  if (asset) {
+    const cashWei = convertTokensToWei({
+      value: asset.cashTokens,
+      token: asset.vToken.underlyingToken,
+    }).toFixed();
+    const borrowBalanceWei = convertTokensToWei({
+      value: asset.borrowBalanceTokens,
+      token: asset.vToken.underlyingToken,
+    }).toFixed();
+    const reservesWei = convertTokensToWei({
+      value: asset.reserveTokens,
+      token: asset.vToken.underlyingToken,
+    }).toFixed();
+    const utilizationRateParams = [cashWei, borrowBalanceWei, reservesWei];
+
+    if (isIsolatedPoolMarket) utilizationRateParams.push(badDebtWei);
+
+    calls.push({
+      reference: 'utilizationRate',
+      methodName: 'utilizationRate',
+      methodParameters: utilizationRateParams,
+    });
```

Comment:
As a best practice, we've been avoiding nesting IF statements in the codebase.
I can suggest one way to rewrite this:

```suggestion
    const utilizationRateParams = [cashWei, borrowBalanceWei, reservesWei];

    calls.push({
      reference: 'utilizationRate',
      methodName: 'utilizationRate',
      methodParameters: isIsolatedPoolMarket ? [...utilizationRateParams, badDebtWei] : utilizationRateParams,
    });
```

Reply:
Nice, thank you